services/QuillLMS/app/workers/reset_demo_account_worker.rb

```diff
@@ -3,10 +3,15 @@
 class ResetDemoAccountWorker
   include Sidekiq::Worker
   sidekiq_options queue: SidekiqQueue::LOW
+  STAFF_DEMO_NAME = "demoteacher+staff"
 
   def perform
+    # sending nil destroys the default /demo account
```

Comment:
As far as I can tell, the `name` field (here)[https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/app/services/demo/report_demo_creator.rb#L224] basically is serving as a proxy for email (i.e. it isn't being used).  Why not update the interface to just take in `email` instead of name for both of these workers and then extract `hello+demoteacher@quill.org` into this worker alongside `STAFF_DEMO_EMAIL`

Reply:
Sure, I'll do this.